frontend/src/pages/Community.vue

```diff
@@ -0,0 +1,218 @@
+<template>
+  <div class="community-page">
+    <div>
+      <b-tabs v-model="tabIndex" content-class="mt-3" align="center">
+        <b-tab :title="$t('community.writing')" active>
+          <contribution-form
+            @set-contribution="setContribution"
+            @update-contribution="updateContribution"
+            @delete-contribution="deleteContribution"
+            v-model="form"
+          />
+        </b-tab>
+        <b-tab :title="$t('community.myContributions')">
+          <contribution-list
+            :items="items"
+            @update-list-contributions="updateListContributions"
+            @update-contribution-form="updateContributionForm"
+            :contributionCount="contributionCount"
+            :showPagination="true"
+            :pageSize="pageSize"
+          />
+        </b-tab>
+        <b-tab :title="$t('navigation.community')">
+          <contribution-list
+            :items="itemsAll"
+            @update-list-contributions="updateListAllContributions"
+            @update-contribution-form="updateContributionForm"
+            :contributionCount="contributionCountAll"
+            :showPagination="true"
+            :pageSize="pageSizeAll"
+          />
+        </b-tab>
+      </b-tabs>
+    </div>
+  </div>
+</template>
+<script>
+import ContributionForm from '@/components/Contributions/ContributionForm.vue'
+import ContributionList from '@/components/Contributions/ContributionList.vue'
+import { createContribution, updateContribution, deleteContribution } from '@/graphql/mutations'
+import { listContributions, listAllContributions, verifyLogin } from '@/graphql/queries'
+
+export default {
+  name: 'Community',
+  components: {
+    ContributionForm,
+    ContributionList,
+  },
+  data() {
+    return {
+      tabIndex: 0,
+      items: [],
+      itemsAll: [],
+      currentPage: 1,
+      pageSize: 25,
+      pageSizeAll: 25,
+      contributionCount: 0,
+      contributionCountAll: 0,
+      form: {
+        id: null,
+        date: '',
+        memo: '',
+        amount: '',
+      },
+    }
+  },
+  methods: {
+    setContribution(data) {
+      this.$apollo
+        .mutate({
+          fetchPolicy: 'no-cache',
+          mutation: createContribution,
+          variables: {
+            creationDate: data.date,
+            memo: data.memo,
+            amount: data.amount,
+          },
+        })
+        .then((result) => {
+          this.toastSuccess(result.data)
```

Comment:
Please toast a message and not the result.

Reply:
ok